network/phonebook/phonebook.go

```diff
@@ -29,23 +29,84 @@ import (
 // of how many addresses the phonebook actually has. ( with the retry-after logic applied )
 const getAllAddresses = math.MaxInt32
 
-// PhoneBookEntryRoles defines the roles that a single entry on the phonebook can take.
+// RoleSet defines the roles that a single entry on the phonebook can take.
 // currently, we have two roles : relay role and archival role, which are mutually exclusive.
```

Comment:
Are they still mutually exclusive? I thought the motivation for this is maintaining multiple roles on a single address?

Reply:
they are not, old comment.